doc/new-book/routing.adoc

```diff
@@ -575,6 +575,20 @@ connector {
 For information about additional attributes, see link:{qdrouterdConfManPageUrl}#_connector[connector] in the `qdrouterd.conf` man page.
 --
 
+. If you want clients to send local transactions to the broker, create a link route for the transaction coordinator:
```

Comment:
One thing I think needs to be clear is that routing the coordinator really only works if there is only one broker with which any transactional interactions are undertaken.